github-orgs/kubeflow/org.yaml

```diff
@@ -267,6 +268,13 @@ orgs:
             - rpasricha
             - prodonjs
             privacy: closed
+          Huawei:
```

Comment:
Please remove teams. We don't set teams in this file.

Reply:
Removed.  So where can we set teams and the teams in this file can be removed?

Reply:
We don't really use teams for anything. Is there a reason you need a team?
We use the company in the user's profile to track affiliation so you should ask your contributors to ensure Huawei there so that their contributions count towards Huawei.

Reply:
Got it, Thanks for your clarify for this.